recipes/iterative-solver/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "iterative-solver" %}
+{% set version = "0.10.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/molpro/iterative-solver/archive/refs/tags//{{ version }}.tar.gz
+  sha256: ee580ceb2f0c65178343f0b3bf4c8703cdc40839aae69cd890ce8a08af1ee162
+
+build:
+  skip: True  # [win]
+  script:
+    - echo __version__ = "{{ version }}" > python/iterative_solver/_version.py
+    - {{ PYTHON }} -m pip install -e python -vv
+  number: 0
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython >=3.0                                 # [build_platform != target_platform]
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - {{ compiler('cxx') }}
+    - cmake >=3.14
+    - make
+    - mpich
+    - numpy >=1.17
+    - eigen
+    - liblapacke
+    - libblas
```

Comment:
All this should move to `host:`. Please also unpin numpy, we're far beyond 1.17 anyway.